src/lib/drivers/extension.js

```diff
@@ -80,6 +80,21 @@ class ExtensionDriver extends Driver {
     this._listeners[eventName].push(cb);
   }
 
+  /**
+   * Bind a one-time listener for protocol events. Listener is removed once it
+   * has been called.
+   * @param {!string} eventName
+   * @param {function(...)} cb
+   */
+  once(eventName, cb) {
+    const cbGuard = function() {
+      cb(...arguments);
+      this.off(eventName, cbGuard);
+    }.bind(this);
```

Comment:
once we move to node 6 we can use an arrow function here and drop the bind (arrow functions don't have access to `arguments` so need rest parameters for this)